symplyphysics/laws/quantum_mechanics/harmonic_oscillator/wave_eigenfunctions.py

```diff
@@ -0,0 +1,65 @@
+from sympy import Eq, Rational, sqrt, exp, factorial, pi
+from sympy.functions.special.polynomials import hermite
+from sympy.physics.units import hbar
+from symplyphysics import (
+    Quantity,
+    Symbol,
+    validate_input,
+    validate_output,
+    units,
+    dimensionless,
+    angle_type,
+    symbols,
+    clone_symbol,
+)
+
+# Description
+## The time-independent Schrödinger equation for the harmonic potential can be solved using the
+## spectral method in the basis of Hermite functions.
+
+# Law: psi_n(x) = 1 / sqrt(2**n * factorial(n))
+#               * ((m * w) / (pi * hbar))**(1 / 4)
+#               * exp(-1 * (m * w) / (2 * hbar) * x**2)
+#               * H_n(sqrt((m * w) / hbar) * x)
+## psi_n - n-th wave eigenfunction
+## n - mode number (n = 0, 1, 2, ...)
+## m - mass of oscillating quantum particle
+## w - angular frequency of quantum oscillator
+## hbar - reduced Planck constant
+## H_n - n-th (physicists') Hermite polynomial
```

Comment:
What `physicists'` means here?

Reply:
There are two types of Hermite polynomials, physicists' and probabilists'. I think it's important to let the user know which ones are used here, to avoid any confusion.